addons/project/models/project_task_recurrence.py

```diff
@@ -81,13 +81,15 @@ def _get_recurrence_delta(self):
 
     def _create_next_occurrence(self, occurrence_from):
         self.ensure_one()
-        if self.repeat_type == 'until' and fields.Date.today() > self.repeat_until:
-            return
```

Comment:
if we remove this, doesn't it mean that if we could create a recurrence in the past before today? when before we would not create anything it seems (but possibly it would still not create it if we only create the next meeting after the previous one (since the previous one should be in the past))

Reply:
I guess keeping this check could make sense as creating tasks in the past is rather useless, but that does not seem functionally invalid to me (Maybe if they were late to deliver they would still want the recurring task to be created, idk). Let's see what the review team has to say.